plugins/ec2/instanceMaxCount.js

```diff
@@ -0,0 +1,94 @@
+var async = require('async');
+var helpers = require('../../helpers');
+
+module.exports = {
+	title: 'EC2 Max Instances',
+	category: 'EC2',
+	description: 'Checks for the number of running instances in an account and triggers a failing result if it exceeds a certain count',
+	more_info: 'It is recommended not to use the default key to keep track of the number of running instances, to prevent unauthorized launch and running excessive costs under your AWS account',
+	link: 'https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/monitoring_ec2.html',
+	recommended_action: 'Go to the ec2 dashboard and audit the instances running, apparently additional unauthorized instances have been launched.',
+	apis: ['EC2:describeInstances'],
+	settings: {
+        instance_count_global_threshold: {
+            name: 'Instance Count Global Threshold',
+            description: 'Checks for the number of running instances globally and triggers a failing result if it exceeds the specified count',
+            regex: '^[1-2]{1}[0-9]{0,2}$',
+            default: 200
+        },
+		instance_count_region_threshold: {
+			name: 'Instance Count Region Threshold',
+			description: 'Checks for the number of running instances in each region and triggers a failing result if it exceeds the specified count',
+			regex: '^[1-2]{1}[0-9]{0,2}$',
+			default: 100
+		}
+	},
```

Comment:
@Mav55 - for this, can we change it to be per-region based? In other words:
```
instance_count_region_threshold: {
		name: 'Instance Count Region Threshold',
		description: 'Checks for the number of running instances in each region and triggers a failing result if it exceeds the specified count. If a region is not set, the region count default will be used. Format: region-name:count. Example: us-east-1:10,us-west-2:20',
		regex: '^[a-z-1-3:,]{5}$',
		default: 'us-east-1:100,us-west-1:100' // ADD ALL REGIONS
	}
}
```
Then, in the `run` below, do a `split` on `,` and parse out the applicable names for each region. Fall back to 100 if the value is invalid.